pylint/checkers/variables.py

```diff
@@ -2715,7 +2716,7 @@ def _check_module_attrs(
                 if module is astroid.Uninferable:
                     return None
             except astroid.NotFoundError:
-                if module.name in self._ignored_modules:
+                if is_module_ignored(f"{module.qname()}.{name}", self._ignored_modules):
```

Comment:
I don't understand what is done here, would you mind explaining ? Is there a problem with the ``module.qname()`` ? The fix probably needs to be in astroid directly if that's the case.

Reply:
In this function we're checking for `no-name-in-module` errors, e.g. `from foo import bar` where `bar` is not found. In this case `module.qname()` is "foo" and `name` is "bar", and since `bar` may be a module we need to check if "foo.bar" matches the ignored modules.

Reply:
All right, that definitely need to be added as a comment 😄 

```suggestion
                # We're checking for no-name-in-module errors, e.g. 'from foo import bar' where bar is not found. 
                # In this case module.qname() is "foo" and name is "bar", and since bar may be a module we need
                # to check if "foo.bar" matches the ignored modules.
                if is_module_ignored(f"{module.qname()}.{name}", self._ignored_modules):
```